cmd/wksctl/init/init.go

```diff
@@ -63,6 +67,10 @@ func multiLineRegexp(pattern string) *regexp.Regexp {
 }
 
 func init() {
+	Cmd.Flags().StringVar(
+		&initOptions.footlooseIP, "footloose-ip", "172.17.0.1", "address of footloose server on host")
```

Comment:
Since we pass this now we probably don't need the default and I wonder if it could cause an issue if someone doesn't pass it.